draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -110,6 +110,7 @@
 <t>
    This document obsoletes RFC 2818, RFC 7231, RFC 7232, RFC 7233,
    RFC 7235, RFC 7538, RFC 7615, RFC 7694, and portions of RFC 7230.
+   Furthermore, it updates RFC 3864.
```

Comment:
Though I still think placing this stuff in the Abstract is a crime against humanity, I would slightly prefer
```suggestion
   This document updates RFC 3864 and
   obsoletes RFC 2818, RFC 7231, RFC 7232, RFC 7233,
   RFC 7235, RFC 7538, RFC 7615, RFC 7694, and portions of RFC 7230.
```

Reply:
+1 to that, and the distaste for doing this sort of thing in the abstract.